system/autoware_hazard_status_converter/src/converter.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 The Autoware Contributors
+// Copyright 2025 The Autoware Contributors
```

Comment:
Please keep the copyright year. This is the publication year.

Reply:
Fixed by [this commit](https://github.com/autowarefoundation/autoware.universe/pull/9971/commits/4244cfa3421b83746294dc9721b880bc19e5b473)